services/movie_session.py

```diff
@@ -1,22 +1,28 @@
+import datetime
+from typing import Optional
+
 from django.db.models import QuerySet
 
 from db.models import MovieSession
 
 
 def create_movie_session(
-    movie_show_time: str, movie_id: int, cinema_hall_id: int
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
 ) -> MovieSession:
     return MovieSession.objects.create(
         show_time=movie_show_time,
-        movie_id=movie_id,
         cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id
     )
 
 
-def get_movies_sessions(session_date: str = None) -> QuerySet:
+def get_movies_sessions(session_date: Optional[str] = None) -> QuerySet:
     queryset = MovieSession.objects.all()
+
     if session_date:
-        queryset = queryset.filter(show_time__date=session_date)
+        queryset = MovieSession.objects.filter(show_time__date=session_date)
```

Comment:
Why did you change this? As for me previous was okay

Reply:
I didn't mean to replace exactly this fragment) just replaced all moviesession.py with my code to avoid mistakes in the annotation concerning Optional:) I'll fix it right now.